### PR TITLE
doc/rados: edit stretch-mode.rst

### PR DESCRIPTION
**Pull request by zdover23**
Refine and supplement the introductory and explanatory text at the top of the /doc/rados/operations/stretch-mode.rst file.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rados/operations/stretch-mode.rst [hunk 1]
```diff
@@ -7,33 +7,47 @@ Stretch Clusters
...
+In the two-site configuration, Ceph expects each of the sites to hold a copy of
+the data, and Ceph also expects there to be a third site that has a tiebreaker
+monitor. This tiebreaker monitor picks a winner if the network connection fails
+and both data centers remain alive.
+
+The tiebreaker monitor can be a VM. It can also have high latency relative to
+the two main sites.
+
+In the three-site configuration, Ceph expects each site to have a copy of the
+data and for there to be an equal number of monitors in every site.
```
Comment (anthonyeleven) [inline doc/rados/operations/stretch-mode.rst:40]: This topology is new to me.  I thought stretch mode wanted OSDs in exactly two sites.

File: doc/rados/operations/stretch-mode.rst [hunk 2]
```diff
@@ -7,33 +7,47 @@ Stretch Clusters
...
+
+The standard Ceph configuration is able to survive MANY network failures or
+data-center failures without ever compromising data consistency. If enough Ceph
```
Comment (jdurgin) [inline doc/rados/operations/stretch-mode.rst:43]: "without ever compromising data consistency" suggests that data consistency is compromised at some point. This is incorrect - it's data availability that is compromised, ceph always keeps consistency (in CAP theorem terms, it's a CP system).

File: doc/rados/operations/stretch-mode.rst (hunk 1 above)
Comment (jdurgin) [reply doc/rados/operations/stretch-mode.rst:40]: Yes I'd leave three-site configuration out of this page, the only use for a 3rd site with stretch mode today is to run one monitor, with no other daemons.

File: doc/rados/operations/stretch-mode.rst (hunk 2 above)
Comment (anthonyeleven) [reply doc/rados/operations/stretch-mode.rst:43]: I agree, s/consistency/availability/  perhaps

File: doc/rados/operations/stretch-mode.rst (hunk 1 above)
Comment (anthonyeleven) [reply doc/rados/operations/stretch-mode.rst:40]: I'd wondered if Zac learned something from Greg that I wasn't aware of.

File: doc/rados/operations/stretch-mode.rst (hunk 2 above)
Comment (zdover23) [reply doc/rados/operations/stretch-mode.rst]: Changed.

File: doc/rados/operations/stretch-mode.rst (hunk 1 above)
Comment (zdover23) [reply doc/rados/operations/stretch-mode.rst]: This was just a rewrite of an already extant sentence.

**Review by anthonyeleven — APPROVED** [review]

**Pull request merged**